db/install.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<XMLDB PATH="admin/tool/heartbeat/db" VERSION="2023110701" COMMENT="XMLDB file for Moodle tool/heartbeat">
+    <TABLES>
+        <TABLE NAME="heartbeat_overrides" COMMENT="Exempted checks for heartbeat tool">
+            <FIELDS>
+                <FIELD NAME="id" TYPE="int" LENGTH="10" NOTNULL="true" UNSIGNED="true" SEQUENCE="true"/>
+                <FIELD NAME="name" TYPE="char" LENGTH="255" NOTNULL="true" UNSIGNED="true" SEQUENCE="false"/>
+                <FIELD NAME="override" TYPE="char" LENGTH="255" NOTNULL="true" UNSIGNED="true" SEQUENCE="false"/>
+                <FIELD NAME="note" TYPE="char" LENGTH="255" NOTNULL="true" UNSIGNED="true" SEQUENCE="false"/>
+                <FIELD NAME="url" TYPE="char" LENGTH="255" NOTNULL="true" UNSIGNED="true" SEQUENCE="false"/>
+                <FIELD NAME="mute_until" TYPE="datetime" NOTNULL="true" UNSIGNED="true" SEQUENCE="false"/>
+                <FIELD NAME="is_resolved" TYPE="int" LENGTH="1" NOTNULL="true" UNSIGNED="true" SEQUENCE="false"/>
```

Comment:
lets swap this from a flag to a date of when it was revolved and if its null it is unresolved